test/lib/compilation/compile-contact-summary.spec.js

```diff
@@ -17,9 +17,6 @@ const genMocks = () => ({
 });
 
 describe('compile-contact-summary', () => {
-  beforeEach(() => sinon.useFakeTimers(1));
-  afterEach(() => sinon.restore());
```

Comment:
FYI @garethbowen this is what was causing the test to hang... Webpack calls `setImmediate` at some point during the compilation and since it was overridden by this mock, it never called its callback.
I don't know why these mocks were necessary in the first place, it looks like a leftover from a previous implementation :man_shrugging:

Reply:
Oh great find! Thank you. That totally makes sense.